source/plugins/go/src/extension/extension.go

```diff
@@ -85,15 +103,25 @@ func getDataTypeToStreamIdMapping() (map[string]string, error) {
 
 	var extensionData TaggedData
 	json.Unmarshal([]byte(responseObjet.TaggedData), &extensionData)
-
 	extensionConfigs := extensionData.ExtensionConfigs
 	logger.Printf("Info::mdsd::build the datatype and streamid map -- start")
 	for _, extensionConfig := range extensionConfigs {
+		logger.Printf("extensionConfig value %s", extensionConfig)

```

Comment:
Remove after testing. Log files will be bloated with the extensionConfig.

Reply:
removed